packages/drivekit_core/flutter_drivekit_core/example/lib/widgets/sliver_start_trip.dart

```diff
@@ -0,0 +1,28 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_drivekit_trip_analysis/flutter_drivekit_trip_analysis.dart';
+import 'package:sliver_tools/sliver_tools.dart';
+
+class SliverStartTrip extends StatefulWidget {
+  const SliverStartTrip({super.key});
+
+  @override
+  State<SliverStartTrip> createState() => _SliverStartTripState();
+}
+
+class _SliverStartTripState extends State<SliverStartTrip> {
+
+  @override
+  Widget build(BuildContext context) {
+    final driveKitTripAnalysis = DrivekitTripAnalysis();
+    return MultiSliver(
+      children: [
+        SliverToBoxAdapter(
```

Comment:
If you have only one Sliver you don't need to wrap it in a `MultiSliver`, here you can directly return the `SliverToBoxAdapter`

Reply:
Fixed, thanks for the feedback